frontend/views/containers/proposals/ProposalBox.vue

```diff
@@ -2,10 +2,10 @@
 li.c-wrapper
   user-image.c-avatar(:username='proposal.meta.username')
   .c-header
-    h4.has-text-bold(data-test='title') {{title}}:
-    span {{humanDate}}
+    h4.has-text-bold(data-test='title') {{ title }}:
+    span {{ humanDate }}
```

Comment:
(Commenting here because I can't comment on unmodified code)

Looking at the implementation of the `humanDate`, I see that it's implemented incorrectly, since it modifies the time using the offset of the date itself. What it needs to do is modify the offset based on the user's OS timezone.

For converting `meta.createdDate` (which is in UTC) to local time, you create a filter instead, and then do:

```
{{ meta.createdDate | toLocalDate }}
{{ meta.createdDate | toLocalDateShort }}

etc...
```

Reply:
I'll improve the code to use filters when I cross paths with this again during the hackathon. 

FYI `getTimezoneOffset` is [based on the user's OS timezone](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset).